src/checks.rs

```diff
@@ -436,6 +440,9 @@ impl CheckCode {
             CheckCode::C404 => CheckKind::UnnecessaryListComprehensionDict,
             CheckCode::C405 => CheckKind::UnnecessaryLiteralSet("<list/tuple>".to_string()),
             CheckCode::C406 => CheckKind::UnnecessaryLiteralDict("<list/tuple>".to_string()),
+            CheckCode::C408 => {
+                CheckKind::UnnecessaryCollectionCall("<dict/list/tuple>".to_string())
```

Comment:
@charliermarsh How would you name this rule?

Reply:
I think what you have here is reasonable.